ios/FluentUI/Vnext/Shimmer/ShimmerView.swift

```diff
@@ -5,125 +5,9 @@
-    /// The alpha value of the view other than the gradient if shimmer is concealing shimmer
-    @objc open var shimmerAlpha: CGFloat {
-        didSet {
-            setNeedsLayout()
-        }
-    }
-
-    /// the width of the gradient in the animation
-    @objc open var shimmerWidth: CGFloat = defaultWidth {
-        didSet {
-            setNeedsLayout()
-        }
-    }
-
-    /// Angle of the direction of the gradient, in radian. 0 means horizontal, Pi/2 means vertical.
-    @objc open var shimmerAngle: CGFloat = defaultAngle {
-        didSet {
-            setNeedsLayout()
-        }
-    }
-
-    /// Speed of the animation, in point/seconds.
-    @objc open var shimmerSpeed: CGFloat = defaultSpeed {
-        didSet {
-            setNeedsLayout()
-        }
-    }
-
-    /// Delay between the end of a shimmering animation and the beginning of the next one.
-    @objc open var shimmerDelay: TimeInterval = defaultDelay {
-        didSet {
-            setNeedsLayout()
-        }
-    }
-
-    /// Tint color of the view if shimmer is revealing shimmer
-    /// Tint color of the middle of the gradient if shimmer is concealing shimmer
-    @objc open var viewTintColor: UIColor = Colors.Shimmer.tint {
-        didSet {
-            setNeedsLayout()
-        }
-    }
-
-    /// Corner radius on each view.
-    @objc open var cornerRadius: CGFloat = defaultCornerRadius {
-        didSet {
-            setNeedsLayout()
-        }
-    }
-
-    /// Corner radius on each UILabel. Set to  0 to disable and use default `cornerRadius`.
-    @objc open var labelCornerRadius: CGFloat = defaultLabelCornerRadius {
-        didSet {
-            setNeedsLayout()
-        }
-    }
-
-    /// True to enable shimmers to auto-adjust to font height for a UILabel -- this will more accurately reflect the text in the label rect rather than using the bounding box.
-    /// `labelHeight` will take precendence over this property.
-    @objc open var usesTextHeightForLabels: Bool = defaultUsesTextHeightForLabels {
-        didSet {
-            setNeedsLayout()
-        }
-    }
-
-    /// If greater than 0, a fixed height to use for all UILabels. This will take precedence over `usesTextHeightForLabels`. Set to less than 0 to disable.
-    @objc open var labelHeight: CGFloat = defaultLabelHeight {
-        didSet {
-            setNeedsLayout()
-        }
-    }
-
-    /// Determines whether we shimmer the top level subviews, or the leaf nodes of the view heirarchy
-    @objc open var shimmersLeafViews: Bool = defaultShimmersLeafViews {
-        didSet {
-            setNeedsLayout()
-        }
-    }
-
-    /// Determines whether the shimmer is a revealing shimmer or a concealing shimmer
-    @objc open var shimmerStyle: ShimmerStyle {
-        didSet {
-            setNeedsLayout()
-        }
-    }
+open class ShimmerView: UIView, TokenizedControlInternal, ControlConfiguration {
```

Comment:
Shouldn't TokenizedControlInternal be enough? I thought ControlConfiguration was for our SwiftUI controls

Reply:
Correct, `ControlConfiguration` should not be used here.